cg/models/lims/sample.py

```diff
@@ -8,65 +8,66 @@
 
 class Udf(BaseModel):
     application: str
-    capture_kit: str | None
-    collection_date: str | None
-    comment: str | None
-    concentration: str | None
-    concentration_sample: str | None
-    concentration_ng_ul: str | None
+    capture_kit: str | None = None
+    collection_date: str | None = None
+    comment: str | None = None
+    concentration: float | None = None
+    concentration_ng_ul: str | None = None
+    concentration_sample: float | None = None
+    control: str | None = None
     customer: str
-    control: str | None
-    data_analysis: str | None
-    data_delivery: str | None
-    elution_buffer: str | None
-    extraction_method: str | None
+    data_analysis: str | None = None
+    data_delivery: str | None = None
+    elution_buffer: str | None = None
+    extraction_method: str | None = None
     family_name: str | None = "NA"
-    formalin_fixation_time: str | None
-    index: str | None
-    index_number: str | None
-    lab_code: str | None
-    organism: str | None
-    organism_other: str | None
-    original_lab: str | None
-    original_lab_address: str | None
-    pool: str | None
-    post_formalin_fixation_time: str | None
-    pre_processing_method: str | None
-    primer: str | None
+    formalin_fixation_time: float | None = None
+    index: str | None = None
+    index_number: str | None = None
+    lab_code: str | None = None
+    organism: str | None = None
+    organism_other: str | None = None
+    original_lab: str | None = None
+    original_lab_address: str | None = None
+    pool: str | None = None
+    post_formalin_fixation_time: float | None = None
+    pre_processing_method: str | None = None
+    primer: str | None = None
     priority: str = Priority.standard.name
-    quantity: str | None
-    reference_genome: str | None
-    region: str | None
-    region_code: str | None
+    quantity: float | None = None
```

Comment:
This value will populate the text UDF "Quantity" in LIMS. It could be good to test that it doesn't break something